.circleci/config.yml

```diff
@@ -0,0 +1,13 @@
+version: 2 
+jobs:
+  build:
+    docker:
+      - image: circleci/openjdk:8-jdk-browsers
+    steps:
+      - checkout
+      - run: mvn package
```

Comment:
mvn package won't give us what we want (an executable jar)

we need to use `mvn -q assembly:single -pl :generator` to generate and executable jar

Reply:
@mrspaceman  this is about building and running the tests, not - at this stage - about building a release. We may do that in future, but not for now

Reply:
so we need to execute `mvn clean test` instead then ?